grml-debootstrap

```diff
@@ -996,9 +996,14 @@ format_efi_partition() {
 # check for EFI support or try to enable it {{{
 efi_support() {
   local efivars_loaded=false
+  # this is for kernels versions before v3.10, which didn't provide efivarfs yet
   if modprobe efivars &>/dev/null ; then
     efivars_loaded=true
   fi
+  # kernel versions v3.10 and newer usually provide efivarfs
```

Comment:
3.10 or 5.10? ;)

Reply:
Yeah it's really 3.10 (I'm really bad at not providing backwards compatibility! :)).
5.10 only changed in debian as such that efivars is no longer available ;(

Reply:
Ah okay, the commit message miss-led me

Reply:
No worries at all and thanks for checking! :)